net.sf.VICE.yml

```diff
@@ -112,3 +110,7 @@ modules:
       
       - type: file
         path: net.sf.VICE-64.png
+
+      - type: shell
+        # Stop using wrong libdir
+        commands: [ sed 's/lib64/lib/g' -i vice/configure.proto ]
```

Comment:
@Compyx It would be nice if this were removed upstream. Hardcoding `lib64` is just a wrong assumption and it should trust the libdir it is given.